conductr/tasks/install-agent.yml

```diff
@@ -0,0 +1,83 @@
+---
+- name: Copy ConductR Agent
+  copy: src=conductr/files/{{ CONDUCTR_AGENT_PKG }} dest={{ REMOTE_WORK_DIR }}
```

Comment:
@edwardcallahan - introduced `REMOTE_WORK_DIR` - this is used as part of ConductR core install.